File: 30 Days of Code/C/Day6_Review.c
#include <stdio.h>
#include <string.h>
#include <math.h>
#include <stdlib.h>

int main() {

    /* Enter your code here. Read input from STDIN. Print output to STDOUT */    
    int n;
    scanf("%d", &n);
    char str[10001];
    
    while(n){
        scanf("%s", str);
        for(int i=0; i<strlen(str); i++){
            if (i%2 == 0){
                printf("%c", str[i]);
            }
        }
        printf(" ");
        for(int i=0; i<strlen(str); i++){
            if(i%2!=0){
                printf("%c", str[i]);
            }
        }
        printf("\n");
        n--;
    }
    
    
    return 0;
}

File: 30 Days of Code/Ruby/Day6_Review.rb
# Enter your code here. Read input from STDIN. Print output to STDOUT
n = gets.chomp.to_i

for i in 0...n do
    s = gets.chomp
    even = ""
    odd = ""
    for j in 0...s.size do
        if j.even?
            even[even.size, 0] = s[j]
        else
            odd[odd.size, 0]= s[j]
        end
    end
    puts even + " " + odd
end